src/PantheonTerminalWindow.vala

```diff
@@ -71,6 +71,7 @@ namespace PantheonTerminal {
         public const string ACTION_SELECT_ALL = "action_select_all";
         public const string ACTION_OPEN_IN_FILES = "action_open_in_files";
         public const string ACTION_SCROLL_TO_LAST_COMMAND = "action_scroll_to_last_command";
+        public const string ACTION_OPEN_IN_BROWSER = "action_open_in_browser";
```

Comment:
Should not use underscores in action names (I know the existing ones need changing) although only affects certain functions.